.github/dependabot.yaml

```diff
@@ -8,4 +8,9 @@ updates:
   allow:
   - dependency-name: "github.com/gardener/gardener"
   labels:
```

Comment:
As we discussed, let's remove the label if it is not causing any troubles for gardener-robot. dependabot already adds its own labels such as `dependencies` and `go`.